cakemail/Tidy-API
# Patch/vulnerabilities

Addresses all vulnerabilities for this project

Upgrades base image for Dockerfile to accommodate updated gems. Changes were also made to apache config file to account for migration to apache 2.4

To test:

1. run `docker build .`
2. run `docker run -d -p 4567:80 <containerId>`
3. navigate to http://localhost:4567/clean
4. Enter the following sample html 

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <style>

    </style>
</head>
<body>
</body>
</html>


```
Should return

`{
    "html": "<!DOCTYPE html>\n<html lang=\"en\">\r\n<head>\n<meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\">\r\n    <meta charset=\"UTF-8\">\r\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\r\n    <title>Document</title>\r\n    <style> TEST </style>\r\n</head>\r\n<body>\r\n</body>\r\n</html>\n"
}`
